src/System.Xml.XPath.XmlDocument/tests/Common/FileHelper.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace XPathTests.Common
+{
+    public static class FileHelper
+    {
+        public static Stream CreateStreamFromFile(string xml)
+        {
+            var xmlPath = Path.GetFullPath(Path.Combine("TestData", xml));
```

Comment:
I tried to get this to work by ensuring the TestData xmls got copied to the output folder and also by using a full path here, but it still doesn't work (the stream is still null).

The original code was:
```cs
var xmlPath = Utils.ResourceFilesPath + xml;
```

Which was generating a string that had some kind of routing:
```
System.Xml.XPath.XmlDocument.Tests.TestData.<FileName>.xml
```

Reply:
Reverted. It's not how it's supposed to work for resource files. More info here: https://github.com/dotnet/maintenance-packages/pull/40#discussion_r1465709445